src/LazyImage/src/BlurHash/BlurHash.php

```diff
@@ -25,7 +25,11 @@
  */
 class BlurHash implements BlurHashInterface
 {
+    /**
+     * @param \Closure(string): string $fetchImageContent
+     */
     public function __construct(
+        private \Closure $fetchImageContent,
```

Comment:
```suggestion
     * @param callable(string): string $fetchImageContent
     */
    public function __construct(
        private callable $fetchImageContent,
```

Reply:
This avoids the need the `\Closure::fromCallable()` stuff in the extension.

Reply:
AFAIK `callable` can not be used as a property type, `Closure` is suggested instead, see https://wiki.php.net/rfc/typed_properties_v2#callable_type

Reply:
Sorry, I should have been more clear, this is what I meant:

```php
    private \Closure $fetchImageContent;

    /**
     * @param callable(string): string $fetchImageContent
     */
    public function __construct(
        ?callable $fetchImageContent = null,
        private ?ImageManager $imageManager = null,
        private ?CacheInterface $cache = null,
    ) {
        if (!$fetchImageContent) {
            $fetchImageContent = fn(string $v) => file_get_contents($v);
        }

        $this->fetchImageContent = $fetchImageContent(...);
    }
```

Reply:
Let's go for it, thanks :)